lms/services/digest.py

```diff
@@ -139,13 +139,29 @@ def instructor_digest(self, h_userid):
                 continue
 
             course_digests.append(
-                {"title": unified_course.title, "num_annotations": num_annotations}
+                {
+                    "title": unified_course.title,
+                    "num_annotations": num_annotations,
+                    "annotators": list(
+                        set(
+                            annotation["author"]["userid"]
+                            for annotation in unified_course.learner_annotations
+                        )
+                    ),
```

Comment:
Adding the list of unique user IDs that annotated in each course. We can use this later to show the number of students who annotated in each course in the email body